.teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt

```diff
@@ -15,5 +15,13 @@ open class BaseGradleBuildType(model: CIBuildModel, val stage: Stage? = null, us
 
     init {
         this.init()
+        cleanup {
```

Comment:
@donat I'm not sure if it's correct to apply this cleanup rule to all build types. Why did you change specific build types to this cleanup rule?

Reply:
It's been a long time ago but I remember some builds produced a ridiculous amount of data and nobody volunteered to clean that up. Without the stricter clean up rules TeamCity disk would be filled up swiftly.

Reply:
OK, let's keep this policy until someone complains.

Reply:
What is the current retention? 14 days?

Reply:
I didn't find the official doc for this, but looking at the current history, it seems to be 30 days.

Reply:
I can see the artifacts of May 29, but not May 26.

Reply:
Do we already keep the artifacts of failed builds for longer? If we do, then a retention of 7 days for successful builds seems fine.

Note that we seem to configure the default cleanup on the `Gradle` project, so if we really want to change retention, we should probably do it there or on the `Check` project, instead of configuring every project.

![image](https://user-images.githubusercontent.com/423186/85859177-1bb88180-b7bd-11ea-969e-ca0b4e2a4a4b.png)